meetings/2021-06-30.md

```diff
@@ -0,0 +1,125 @@
+
+Elf: https://github.com/solid/web-access-control-spec#aclaccesstoclass
+Just to clarify, that would be WAC+ because WAC doesn't support it.
+
+Henry: Of course, but it's there in the ontology so it could be used and because it's uncommonly used, it's harder to implement.
+The accessToClass just needs to be used. That's what you're describing.
+
+Elf: Just saying it's not supported by design in https://github.com/solid/web-access-control-spec
+
+Henry: The spec at the moment is extremely limited but it's not like we don't have to improve it.
+Let's see whether things get blocked or we can slowly move forward.
+Justin speaks of defining sets of resources and that solution fits very nicely.
+
+Justin: The way it's designed, clients should be able to read and write it on common data structures.
+In the current implementation that we've done, we look at data grants and then know how to apply the permissions on resources depending on the structures of resources and data. The authorization agent that understands data structures knows how to apply permissions.
+The authorization server who has access could look at the grant directly and be able to say whether you have access or not.
+Sort of leapfrogging having acp policies applying data grants.
+
+Elf: If you have an algorithm translating the data grants could even not be required.
+If using data grants, you need to be warry of keeping a single source of truth.
+
+Henry: You're bringing a whole bunch of architecture changes. I think it's a little bit like where are the grants... We'll have to look at it and see whether it's better than WAC.
+There's a whole thing about why we may want ACP and a lot of the assumptions for justifying it might have been wrong.
+We should move on to use cases and there, the precise explanation of how things are going, it's difficult to say just by listening to the argument.
+
+Elf: I'm a little worried that the use cases started as WAC UCRs. But Henry, you're right we should compare the solutions. I think it's important to keep things simple from a application development perspective.
+It's a big change of assumptions to have any application have to understand control rules, because I don't think anything that isn't dedicated shouldn't have to care about that.
+
+Henry: I see an authz agent which is on the server and the one on the client. Because the use cases are too close, (two servers owned by the same person), if you just had a server with access it knows everything.
+That's part of what I address in my pull request which is next.
+
+### PR 232 on PR 231: Initial use case for consent workflow
+
+Henry: You have distinction between guards on the server and clients. There should be no entities that has overview of all that, but there's one client that has access to all your verifiable credentials, all your claims and that lives on your phone or computer. The wallet agent is guarding/authorising an app to receive a token. It can decide what apps are allowed to do.
```

Comment:
```suggestion
Henry: You have distinction between guards on the server and clients. There should be no entity that has overview of all that, but there's one client that has access to all your verifiable credentials, all your claims; and that lives on your phone or computer. The wallet agent is guarding/authorising an app to receive a token. It can decide what apps are allowed to do.
```